TLM/PR_REVIEW_INSTRUCTIONS.md

```diff
@@ -0,0 +1,18 @@
+
+## Pull Request Review
+
+
+### Clone PR as new branch inside repository folder:
+
+* get __PR__ index - this number with __#__ after __PR__ name
+* using preferred __git console__ inside project folder ``` \<Folder_of_cloned_repository>\TLM\ ```
```

Comment:
In Git Desktop (Atom-version) the PR can be selected from the 'Current Branch' drop-down.

Reply:
Nice ;)